src/msg/async/rdma/RDMAStack.cc

```diff
@@ -245,7 +247,8 @@ void RDMADispatcher::polling()
       if (!num_qp_conn && done)
         break;
 
-      if ((ceph_clock_now() - last_inactive).to_nsec() / 1000 > cct->_conf->ms_async_rdma_polling_us) {
+      uint64_t now = Cycles::rdtsc();
+      if (Cycles::to_microseconds(now - last_inactive) > cct->_conf->ms_async_rdma_polling_us) {
```

Comment:
@branch-predictor Yes, the problems of RDTSC mentioned by you above are also my concerns.  
 Even though the difference of TSC between CPUs can be solved by pin the polling thread to a specified core, I don't think it is elegant. 
Do you have any other suggestions to reduce the CPU usage of RDMA polling thread?